src/App.tsx

```diff
@@ -7,6 +7,18 @@ import MyClowder from './Components/MyClowder';
 import OverlayWrapper from './Components/Styling/OverlayWrapper';
 import PageWrapper from './Components/Styling/PageWrapper';
 
+const {
+	REACT_APP_MAPBOX_ACCESS_TOKEN,
+	REACT_APP_BACKEND_HOST,
+	REACT_APP_LOGGED_IN_USER,
+} = process.env;
+if (
+	!REACT_APP_BACKEND_HOST ||
+	!REACT_APP_MAPBOX_ACCESS_TOKEN ||
+	!REACT_APP_LOGGED_IN_USER
```

Comment:
What about when we don't want the user to be logged in, so the login screen shows?

Reply:
True - I also thought about the fact that the logged in user would be handled in localStorage, so I think that removing the `REACT_APP_LOGGED_IN_USER` would be the best way to go about it. The issue with this is it would cause errors in Maria's code (relating to [CL-4 PR-15](https://github.com/clowder-cloud/frontend/pull/15)) as we could be requesting to read a user with the username undefined (presumably it will translate to ""). I think this will be resolved when the whole user context and logging in tickets are resolved. You're call about what to do - If you want me to just remove this `REACT_APP_LOGGED_IN_USER` I can, but I think it'd be temporarily unsafe.